### ISSUE TITLE
train_parallel dose not support '--pretrained' option?

### ISSUE THREAD

**Issue by cysin** [deepinsight/insightface#574]
I've noticed that resuming with pre-trained weights is disabled in the code. Is this intended to be so or for some other reasons? Thanks!

**Comment by nttstar** [comment]
Yes right now pretrained option was not supported in train-parrall

**Comment by cysin** [comment]
Thanks for the reply!

**Comment by xizi** [comment]
Do you have a plan to support pretrained model in train-parrall?

**Comment by nttstar** [comment]
yes but later.

**Comment by maryhh** [comment]
https://github.com/deepinsight/insightface/blob/1a8f0fc6b6aad73594c5e8f45704192c4185e168/recognition/parall_module_local_v1.py#L148
why not Initialize arg_params and aux_params here? esym:[fc1,label].asym[fc7],only initialize parameter until fc1.
